src/main/java/teammates/storage/api/FeedbackSessionsDb.java

```diff
@@ -542,41 +541,46 @@ public void deleteFeedbackSessionsForCourses(List<String> courseIds) {
     }
     
     @SuppressWarnings("unchecked")
-    private List<FeedbackSession> getFeedbackSessionEntitiesWithUnsentOpenEmail() {
+    private List<FeedbackSession> getFeedbackSessionEntitiesNeedingOpenEmail() {
         Query q = getPm().newQuery(FeedbackSession.class);
-        q.declareParameters("boolean sentParam, Enum notTypeParam");
-        q.setFilter("sentOpenEmail == sentParam && feedbackSessionType != notTypeParam");
+        q.declareParameters("java.util.Date startTimeParam, boolean sentParam");
+        q.setFilter("startTime > startTimeParam && sentOpenEmail == sentParam");
```

Comment:
Hm won't this also get feedback sessions that have not yet been opened? To me that is somewhat unintuitive as the method name implies that it gets feedback sessions which have already opened.